File: kimdokyun/Python/boj_1676.py
N = int(input())
cnt2 = 0
cnt5 = 0
for i in range(1,N+1):
    while i%2==0:
        cnt2+=1
        i/=2
    while i%5==0:
        cnt5+=1
        i/=5
print(min(cnt2,cnt5))

File: Closed/kimdokyun/C++/boj_1676.cpp
#include<stdio.h>

int main() {
	int N,cnt2=0,cnt5=0;
	scanf("%d",&N);
	for(int i=1;i<=N;i++) {
		int n=i;
		while(n%2==0) {
			cnt2++;
			n/=2;
		}
		while(n%5==0) {
			cnt5++;
			n/=5;
		}
	}
	printf("%d",cnt2>cnt5?cnt5:cnt2);
}
